### PR TITLE
update version number

### PR DESCRIPTION
**Pull request by GodsonLeigh**

### CODE REVIEW

File: app/views/cookies/cookie-policy.njk [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
   {% if updated %}
     {% set html %}
-      <p class="govuk-notification-banner__heading">
+    <p class="govuk-notification-banner__heading">
```
Comment (AbidemiAdio) [inline app/views/cookies/cookie-policy.njk:20]: check indentation please.

File: app/views/cookies/cookie-policy.njk [hunk 2]
```diff
@@ -173,4 +174,4 @@
     </div>
   </div>
 
-{% endblock %}
+{% endblock %}
```
Comment (AbidemiAdio) [inline app/views/cookies/cookie-policy.njk:176]: remove extra line

File: app/views/value.njk [hunk 3]
```diff
@@ -31,4 +32,4 @@
       </div>
     </div>
   </div>
-{% endblock %}
+{% endblock %}
```
Comment (AbidemiAdio) [inline app/views/value.njk:35]: as above.

File: app/views/cookies/cookie-policy.njk (hunk 1 above)
Comment (GodsonLeigh) [reply app/views/cookies/cookie-policy.njk:20]: This indentation is set by Javascript standard, if I overwrite it and indent, the spacing is removed when I save.

File: app/views/value.njk (hunk 3 above)
Comment (GodsonLeigh) [reply app/views/value.njk:35]: There is no extra line in the code, though it looks that way on the snippet.

File: app/views/cookies/cookie-policy.njk (hunk 2 above)
Comment (GodsonLeigh) [reply app/views/cookies/cookie-policy.njk:176]: extra line removed

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**